server/routes/index.js

```diff
@@ -37,6 +39,17 @@ router.get("/schema/:schemaId", async (req, res) => {
     }
 });
 
+router.post("/testHook", cors(), async (req, res) => {
+    const { hook } = req.body;
+    try {
+        const hookResponse = await axios.get(hook);
```

Comment:
@josemigallas I used axios over fetch as I need to make a `POST` at https://github.com/aerogear/data-sync-ui/blob/14fbb2e511e5654f4bf296115370cb0928ebe467/ui/components/resolvers/HookFormGroup.js#L35  and it seems a bit tidier than having to add options object for making a post with fetch.

But if that is preferred than I will change to fetch?

Reply:
The sync server uses `POST` for both pre and post hooks. We should use `POST` here also.

Reply:
@cfoskin sorry about the weird requestObject name: https://github.com/aerogear/data-sync-server/blob/master/server/lib/resolvers/wrapResolverWithHooks.js#L8

We have to match up because it could lead to unexpected results. Imagine a case where the server we send the pre and post hooks to is an express server that has its handlers set up like this:

````js
app.post('preHook', (req, res) => {})
app.post('postHook', (req, res) => {})
```

Using GET would result in 404, which would result in an error in the data sync UI. Does that make sense?

Reply:
@darahayes  we want the get to result in a 404 so we can use that in the ui to render error state

Reply:
This is just for a TEST button which tries to hit the url or hook and verifies it exists and nothing else

Reply:
@cfoskin understood, but I think maybe I didn't explain myself well enough. Imagine a case where the end user has set up a custom webhooks server that only accepts `POST` requests because they know that's what the data-sync-server sends.

When you go to press the test button in the data-sync-ui and it makes a `GET` request. The custom webhook server is going to return a 404 and then the UI is going to display an error, saying the request failed, but in reality it would have been successful using a POST. So basically, the UI would be wrongfully showing an error. Does that make sense?

Reply:
@darahayes  Yep that does make sense, I wasn't sure what you meant initially. Ive updated it